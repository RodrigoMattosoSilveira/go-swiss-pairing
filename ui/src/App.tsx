import React from 'react';
import MembersPage from "./members/MembersPage";
import logo from './logo.svg';
import './App.css';

function App() {
       return (
         <div className="container">
                <MembersPage />
             </div>
         );}

export default App;
