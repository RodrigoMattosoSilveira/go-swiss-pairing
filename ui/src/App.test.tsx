import React from 'react';
import App from './App';
import renderer from "react-test-renderer";

describe('<App />', () => {
  // test('renders home heading', () => {
  //     // render(<HomePage/>);
  //     // expect(screen.getByRole('heading')).toHaveTextContent('Home');
  //     test('should render without crashing', () => {
  //         render(<HomePage />);
  //     });
  // });
  test('snapshot', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
