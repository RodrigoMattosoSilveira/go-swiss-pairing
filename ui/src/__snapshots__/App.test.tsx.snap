// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> snapshot 1`] = `
Array [
  <header
    className="sticky"
  >
    <span
      className="logo"
    >
      <img
        alt="logo"
        height="99"
        src="/assets/logo-3.svg"
        width="49"
      />
    </span>
    <a
      aria-current="page"
      className="button rounded active"
      href="/"
      onClick={[Function]}
    >
      <span
        className="icon-home"
      />
      Home
    </a>
    <a
      className="button rounded"
      href="/members"
      onClick={[Function]}
    >
      Members
    </a>
  </header>,
  <div
    className="container"
  >
    <h2>
      Home
    </h2>
  </div>,
]
`;
